src/doc-editors/ServerGroupsEditor.vue

```diff
@@ -421,6 +454,11 @@ export default defineComponent({
     },
 
     ...mapStores(useBranchesStore),
+
+    isDeleteServerGroupDocNameValid(): boolean {
+      const newName = this.deleteServerGroupDocName.trim()
```

Comment:
newName is not right, it's a name for confirmation of deleting the Server Group, not creating a new one

```suggestion
      const deleteConfrimInputName = this.deleteServerGroupDocName.trim()
```